python/activity_stream/widget_note.py

```diff
@@ -225,10 +225,10 @@ def set_note_info(self, data):
         self._set_timestamp(data, self.ui.date)
         
         # set the main note text
-        self.ui.content.setText(data["content"])
+        self.ui.content.setText(data.get("content", ""))
```

Comment:
Should this be `self.ui.content.setText(data["content"] or None)` ? (Shotgun can return None for an empty field.)

To me, it feels like we should ideally be very clear about which fields `data` is assumed to contain (and updating my terrible docstring for `set_note_info` - sorry!!!!) rather than patching.

If we do expect missing key from the dict, it would be awesome if we could mention it in the docstring.

Sorry for being picky and thanks for cleaning this stuff up.